astroplan/core.py

```diff
@@ -310,7 +355,20 @@ class FixedTarget(Target):
     """
     An object that is "fixed" with respect to the celestial sphere.
     """
-
+    def __init__(self, coord, name=None, **kwargs):
```

Comment:
I went to the @adrn [-style extreme](https://github.com/astroplanners/astroplan/pull/4#discussion_r32672422) here and started by initializing a `FixedTarget` with only a `SkyCoord`, because I'm starting to agree that this is the clearest, least redundant way to write the code.